### ISSUE TITLE
[UX] Use "Configure" icons instead of the full word

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#3539]
Now that we have icons in the admin menu, it strikes me that we could use the "Configure" icon instead of a button that spells out the full word. For example, on the Manage Display pages, a view such as this (admittedly only this many fields are likely to have a "Configure" option if you are using `field_formatter_settings` but it makes the point):

![screen shot 2019-02-12 at 11 28 34 am](https://user-images.githubusercontent.com/871421/52655462-d3dfd580-2eb9-11e9-9f50-e7b246792e01.jpg)

With a little modification (CSS only) we could have something more like this:

![screen shot 2019-02-12 at 11 28 14 am](https://user-images.githubusercontent.com/871421/52655490-e22df180-2eb9-11e9-8aba-96be4f190473.jpg)

If people agree I can make a PR for Seven that would do this.

**Comment by klonos** [comment]
I am all for this, but we need to ensure that the icon buttons are still accessible after this change.

Could this be a duplicate of #1974?

**Comment by laryn** [comment]
Definitely related but (maybe) not a duplicate. They should probably be matching, though.

This one can happen with straight CSS and should still be accessible (if I understand it) because the words are still there in the code.

**Comment by klonos** [comment]
> Definitely related but (maybe) not a duplicate.

True that. Most of the discussion and the mockups/ides there are about the Layout UI, so I have renamed that issue accordingly.

**Comment by laryn** [comment]
But this does get more complicated when it is a drop button instead of just a single button...

**Comment by jenlampton** [comment]
> we could use the "Configure" icon instead of a button that spells out the full word

This makes me nervous, for a few reasons. 

For starters, if we do this in one location, It would be best if we did id consistently in all locations where we have drop-buttons. Not all the actions displayed in Drop-buttons in core alone have suitable icons (not to mention anything could be added by any number of contrib modules). If we were to choose an icon at random for something like "manage displays" we'd really be hurting usability as very few people would know that this icon maps to that action.

Additionally, because these are drop-buttons, we'd need to consider what a stack of icons in a drop-down would look like, and how usable that would be on smaller devices, with a limited target area, and larger fingers.

**Comment by stpaultim** [comment]
There is clearly no consensus that this is a good idea yet. However, I'm suggesting this as a good first issue because I think it would be helpful to have someone with fresh eyes look at this issue, look at how it's currently implemented and offer their opinion. You don't need to be an expert in Backdrop CMS to have an opinion on this. Just [spin up a sandbox site](https://backdropcms.org/demo) to look at the current state of things. 

If you like this idea and if it is not too difficult, creating a Pull Request might help us decide if we want to proceed with this issue. But, keep in mind that there is no consensus on this yet, the goal is to help move us towards one.

MY OPINION: My initial thought is that this looks like a good idea, but I'm also concerned about consistency.

**Comment by indigoxela** [comment]
> but I'm also concerned about consistency.

Consistency is one, web accessibility is another.

**Comment by BWPanda** [comment]
I tend to agree with @jenlampton. No point adding an icon for 'Configure' if we don't add it for other similar buttons/actions too.

> With a little modification (CSS only) we could have something more like this:

If it's just a simple CSS change, then perhaps this can be left to themes/contrib...
